spec/waiter_spec.rb

```diff
@@ -29,4 +29,40 @@
       expect(duration).to be_within(0.1).of(timeout)
     end
   end
+
+  describe '.wait_with_capybara (or_raise)' do
+    it 'raises the given error if the block is not truthy' do
```

Comment:
could we have 2 contexts here? with an error defined, and without one defined?

Reply:
I would ordinarily agree, but since each one would contain just one test it's not super helpful.